src/int.cr

```diff
@@ -395,6 +395,36 @@ struct Int
     (self & mask) == mask
   end
 
+  # Returns the number of bits of this int value.
+  #
+  # “The number of bits” means that the bit position of the highest bit
+  # which is different to the sign bit.
+  # (The bit position of the bit 2**n is n+1.)
+  # If there is no such bit (zero or minus one), zero is returned.
+  #
+  # I.e. This method returns `ceil(log2(self < 0 ? -self : self+1))`.
```

Comment:
Example formatted with `crystal tool format`
```suggestion
  # I.e. This method returns `ceil(log2(self < 0 ? -self : self + 1))`.
```

Reply:
I copied this verbatim from Ruby's documentation. But I guess this is fine. Thanks!